data/net/minecraft/network/chat/PlayerChatMessage.mapping

```diff
@@ -0,0 +1,28 @@
+CLASS net/minecraft/network/chat/PlayerChatMessage
+	METHOD <init> (Lnet/minecraft/network/FriendlyByteBuf;)V
+		ARG 1 buffer
+	METHOD filter (Lnet/minecraft/network/chat/FilterMask;)Lnet/minecraft/network/chat/PlayerChatMessage;
+		ARG 1 filterMask
+	METHOD filter (Z)Lnet/minecraft/network/chat/PlayerChatMessage;
+		ARG 1 shouldFilter
+	METHOD hasExpiredClient (Ljava/time/Instant;)Z
+		ARG 1 instant
```

Comment:
```suggestion
		ARG 1 timestamp
```
The class `Instant` stores an exact point in time. I believe `timestamp` would be a better parameter name as I feel it better describes the parameter.

Reply:
I think other places I call it timeStamp.

Reply:
`timeStamp` is also fine.